### PR TITLE
Changes to make APEL parser process and send accounting for HTCondorCE

### PR DESCRIPTION
**Pull request by jcasals**
New pull request fixing the error in the conf file. These are the modified files:

```
apel/db/backends/mysql.py
apel/db/records/__init__.py
apel/parsers/htcondor.py
conf/parser.cfg
apel/db/records/htcondorce.py
apel/parsers/htcondorce.py
apelparser
```

### CODE REVIEW

**Comment by geonmo** [comment]
@jcasals Could you fix this PR asap? I need to build our EGI site in May.

**Comment by gregcorbett** [comment]
@jcasals apologies for the delay in getting back to you on this.

Could you explain why `apel/parsers/htcondor.py` had to be changed to get the CE functionality working?

We had assumed the new CE parser would have been completely separate from the existing HTCondor parser we have.

**Comment by kreczko** [comment]
@gregcorbett Since HTCondor CE is just a special configuration of an HTCondor Scheduler wouldn't it make sense to use the same code?

File: apel/db/records/htcondorce.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+'''
+   Copyright 2012 Konrad Jopek
```
Comment (kjopek) [inline apel/db/records/htcondorce.py]: I guess I am not author of this file :)

Comment (tofu-rocketry) [reply apel/db/records/htcondorce.py]: I'll sort out all the notices before anything's merged to dev.

**Pull request closed without merging**